package.json

```
@@ -59,12 +59,12 @@
     "eslint-config-standard": "^16.0.0"
   },
   "peerDependencies": {
-    "eslint": ">=7.12.1",
-    "eslint-plugin-import": ">=2.22.1",
-    "eslint-plugin-node": ">=11.1.0",
-    "eslint-plugin-promise": ">=4.2.1",
-    "typescript": ">=3.9",
-    "@typescript-eslint/eslint-plugin": ">=4.0.1"
+    "eslint": "^7.12.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "typescript": "^3.9",
```

More common to write:

```suggestion
    "typescript": "^3.9.0",
```